spec/features/photoes_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+feature 'Uploading photoes' do
+
+  scenario 'a not logged in user can\'t upload a photo' do
+    visit '/'
+    click_link 'Upload'
+    expect(current_path).to eq '/users/sign_in'
+  end
+
+  context 'when logged in' do
+    background do
+      visit '/'
+      sign_up(username: 'allimac',email: 'camilla@email.com', password: 'pass1234')
+    end
+
+    scenario 'a user can upload a photo' do
+      visit '/'
+      upload_photo
+      expect(page).to have_css('img', 'city.jpg')
+      expect(page).to have_content('by allimac')
+      expect(current_path).to eq '/photos'
+    end
+  end
+
+end
+
+feature 'Viewing and deleting photoes' do
+
+  context 'when no photos have been uploaded yet' do
+    scenario 'no photos are shown' do
+      visit '/'
+      expect(page).to have_content 'No photos yet!'
+      expect(page).not_to have_css('img')
+    end
+  end
+
+  context 'when there are photos' do
+    background do
+      visit '/'
+      sign_up(username: 'allimac',email: 'camilla@email.com', password: 'pass1234')
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)